docs/usage.md

```diff
@@ -269,6 +269,18 @@ With this configuration in place deployment should happen automatically every ti
 
 There is also a [Shiny server application](https://posit.co/download/shiny-server/), which you can install on your own infrastruture and use to host applications yourself.
 
+## Pathway enrichment analysis
```

Comment:
Could you include GSEA here? At least mention it as an option/ add a subheading so that I can go back and fill it in (I didn't realise I hadn't documented that at all).

Reply:
Will do!